docs/dev_guide/migration/migration-rules.rst

```diff
@@ -56,11 +56,14 @@ Migration rules are specified in YAML files. A single rule file may contain mult
      - Required. Specifies the priority of the rule: ``Takeover`` > ``Default`` > ``Fallback``.
        When there are rule conflicts, the rule with higher priority will take precedence.
    * - Kind
-     - ``Macro`` | ``API`` | ``Header`` | ``Type`` | ``Class`` | ``Enum`` | ``DisableAPIMigration`` | ``PatternRewriter`` | ``CMakeRule``
+     - ``Macro`` | ``API`` | ``Header`` | ``Type`` | ``Class`` | ``Enum`` | ``DisableAPIMigration`` | ``PatternRewriter`` | ``CMakeRule`` | ``PythonRule``
      - Required. Specifies the rule type.
    * - CmakeSyntax
      - String value
      - Required. Specify the CMake syntax name that will be migrated. Use the unique name for the CMake syntax.
+   * - PythonSyntax
+     - String value
+     - Optional. Specify the Python setup syntax name that will be migrated. Use the unique name for the Python syntax.
```

Comment:
it is not python syntax? 
```suggestion
     - Optional. Specify the Python setup syntax name that will be migrated. Use the unique name for the Python setup syntax.
```

Reply:
Changed "Python setup syntax" to "Python syntax"